README.md

```diff
@@ -55,6 +55,13 @@ import marqo
 
 mq = marqo.Client(url='http://localhost:8882')
 
+# Delete index if it exists (to avoid duplicate documents)
```

Comment:
This is correct, but it makes the README more convoluted for first time users, who most likely, do not have an index created.